projects/intro-to-programming/Java101.md

```diff
@@ -416,8 +416,8 @@ public class Main {
 		int index = 0;
 		while (index < 100) {
 			arr[index] = 2 - index % 2;
+			index = index + 1;
```

Comment:
You're correct. However I just noticed that line 399 is also wrong as the loop will never end (not that the code would compile in the first place because `n` is never defined), iteration variable should be `index`

Please fix and resubmit!